Update 2018-10-30-why-you-need-to-implement-a-plugin.md (#7)
Shpota/shpota.github.io: _posts/2018-10-30-why-you-need-to-implement-a-plugin.md

date:   2018-10-30 10:00:00 +0200
comments: true
---
IntelliJ IDEA is a powerful tool that aims at increasing
productivity. It speeds up the development process and optimizes 
most of the standard use cases such as code formatting, code generation, 

> navigation, etc. However, the utility of the IntelliJ IDEA is not limited 

Author: Alright, I can change it to their site
```suggestion
navigation, etc. However, their site is not limited 
```

> navigation, etc. However, the utility of the IntelliJ IDEA is not limited 

Reviewer: You can change it either to "it" or to "IntelliJ IDEA". I was talking about using definitive article "the". In the official site they call their product "IntelliJ IDEA", not "the IntelliJ IDEA". That was the point.

to these processes. For instance, I set up my IDE to generate tests prefixed with 

> the word 'should'. The plugin also automatically [adds annotations to test classes 

Author: ```suggestion
the word 'should'. The IDE (IntelliJ IDEA) tool also automatically [adds annotations to test classes 
```

> the word 'should'. The plugin also automatically [adds annotations to test classes 

Reviewer: Let's leave either "The IDE" or "IntelliJ IDEA" or "It". Clarification in brackets is not needed here, it is clear form the context.

depending on their names](https://stackoverflow.com/a/47202979/2065796). 
But what could make an IDE even more useful is implementing
an own plugin that will optimize processes that cannot be 
customized via standard tools. What exactly could be achieved thereby? 
And how complex is it? This is what I'll try to address in this blog.

{% 
  include picture.html 
  href="2018-10-30-car-automation.jpg" 
  copyright="oemupdate. com"
%}

> If you work within a team for a certain amount of time, you are bound

Author: I can change the phrasing, sure. How about: A team that works together continuously naturally develops ...
```suggestion
A team that works continuously together naturally develops 
```
